service_systemd_linux.go

```diff
@@ -162,6 +162,8 @@ func (s *systemd) Status() error {
 const systemdScript = `[Unit]
 Description={{.Description}}
 ConditionFileIsExecutable={{.Path|cmdEscape}}
+After=network.target
+After=syslog.target
```

Comment:
What's the reason to add this? Does it fix any reproducible bug? This customer was not using systemd, so if this is just a backport/bug fix patch/release for this customer then I wonder why such change. It also diverges from upstream, so would be nice to have strong reason to keep this patch different than upstream - unless you plan to push this to upstream too?